src/Jackalope/Version/VersionHandler.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace Jackalope\Version;
+
+use Jackalope\Session;
+use Jackalope\Transport\AddNodeOperation;
+use Jackalope\Transport\WritingInterface;
+use PHPCR\InvalidItemStateException;
+use PHPCR\NodeInterface;
+use PHPCR\PropertyType;
+use PHPCR\UnsupportedRepositoryOperationException;
+use PHPCR\Util\UUIDHelper;
+use PHPCR\Version\VersionException;
+
+class VersionHandler
```

Comment:
I don't really want to mention Sulu, the previous doc block was just an accident...